### DISCUSSION TITLE
Eventos

### DISCUSSION THREAD

**Discussion by bxbycry** [zebnat/morpho-dimension#34] [Ideas]
Creo que estaría muy bueno el hacer eventos únicos por ciertas fechas, en estos dar ****objetos/premios**** que no se hayan visto antes, esto incitara a que todos quieran participar, Básicamente las recompensas podrían ser ropa exclusiva.

  _1: Evento de San Valentín_
       -El usuario deberá crear una carta, Sea 
        dirigida a amigos/pareja/comunidad
        donde realmente muestre sus 
        sentimientos a base de palabras, esté
        evento haría que los usuarios expresen
        sus emociones de una manera única.

  _2: Evento de Halloween_
       -El usuario deberá contar una historia
        paranormal o inventar una, esto puede
        promover su imaginación a un sin fin
        de límites.

Evidentemente se publicaría al final del evento el ganador, se revelaría el nombre solo si esté lo desea. También comprendo que es mucho tiempo para hacerlos por el tema de tener que revisar uno por uno, En esté caso los interesados en **MD** podríamos ayudar claro está con el fin de ser buen jurado sin tener que privilegiar a sus compañeros.

**Comment by ghost** [comment]
me gusta la idea pero deberias expandirte mas en eso, eventos de parejas en san valentin seria buena idea, tambien para eventos de amigos el mismo dia, formas en que se sientan comodos y no tengan timides
